### ISSUE TITLE
PETSc installation --download-fblaslapack

### ISSUE THREAD

**Issue by jfregin** [firedrakeproject/firedrake#1839]
Hello,

is there any downside installing Blas/Lapack by adding `--download-fblaslapack` to the `petsc_options`? I'm asking because the install instructions on the firedrake webpage ask to have Blas/Lapack installed when installing without package manager. If there is no downside, why is this not the default option? Below is some explanation on how I came up with the question.

I succesfully installed firedrake on openSUSE, however I had a few issues during the installation. Mainly because i failed to get Blas/Lapack running and therefore the PETSc installation failed. Then I tried building PETSc myself with the configure options that firedrake suggests which worked fine (additionally I added `--download-fblaslapack` ). However, for some reason the firedrake installation still failed (I don't have the log files anymore).
I resolved the problem by just adding `--download-fblaslapack` to the `petsc_options` in the firedrake-install script script. Afterwards the installation was successful. 

Kind regards,

Joscha

**Comment by wence-** [comment]
`--download-fblaslapack` gives you a reference version of BLAS/LAPACK (from netlib). While this will work, it is low performance. We therefore recommend that you install a high-performance library providing this (often `openblas` will work). 

We are likely to migrate to installing openblas and getting PETSc to build against that in the new future, so it would be good to know (if you can reproduce) what went wrong with the system provided blas/lapack that you tried.

**Comment by jfregin** [comment]
Thank you. The "system provided" blas/lapack was the one that I installed with PETSc. I can rerun the installation using blas/lapack from netlib today, to see what went wrong. I'd also gladly test the installation with openblas, but it will take a while as I have only user rights on my working machine and the administrator is on holiday. I also tried compiling blas/lapack myself, however only blas seems to work. With lapack I get a segmentation fault and unfortunately the solution to this problem (hopefully) requires my administrator.
